spec/info/description.md

```diff
@@ -18,7 +18,9 @@ Before using this API you need the following:
 * **Zenvia Account**: create an account on [Zenvia platform's site](https://app.zenvia.com/)
 * **Integrations**: configure desired channels to send and/or receive messages on the [integrations page](https://app.zenvia.com/home/credentials)
 * **API Token**: create an API token on the [API console](https://app.zenvia.com/home/api)
-* **Webhook** (optional): receive messages by subscribing to events using [subscriptions API resources](#tag/Subscriptions)
+* **Webhook**: receive messages by subscribing to events using [subscriptions API resources](#tag/Subscriptions)
+    * _Webhook for status_ **(required)**: It's necessary to register the webhooks to keep up with the requests status.
+    * _Webhook for messages_ (optional): 
```

Comment:
O texto do messages ficou no lugar errado:
![image](https://user-images.githubusercontent.com/4666758/173931270-29ea4454-cfaa-4fa9-a734-43fca4a1311f.png)

E algumas sugestões:
- em vez de `required` que não é verdade acho que ficaria melhor algo tipo `important`
- Acho que em vez de usar `for`, como em `Webhook for status` poderia ser `Status webhook` ou mesmo só `Status`. O mesmo pro message.
- A descrição do status acho que poderia dizer o porque, algo tipo `Since our messaging API is asynchronous, it is necessary to register a webhook in order to know whether the message sending was successful or not.`
- Acho que a descrição do webhook de mensagem pode ser quebrado em dois pra ficar o link a linha genérica de webhook. Tipo: `**Webhook**: subscribe to events using [subscriptions API resources](#tag/Subscriptions).` e `_Webhook for messages_ (optional): receive message responses by subscribing to message events.`